packages/angular/src/client/request.headers.interceptor.ts

```diff
@@ -75,6 +75,10 @@ export class RequestHeadersInterceptor implements HttpInterceptor {
                     if (res.headers.has(HTTP_HEADERS.etag)) {
```

Comment:
can you do something like `equalsIgnoreCase` or something what if it changes again to `Etag` for example?

Reply:
Generally it shouldn't change, however if we are going to do that for etags, we might as well introduce a generict solution for the key sensitivity of all headers in separate PR.